app/pods/publications/index/controller.js

```diff
@@ -110,4 +101,44 @@ export default class PublicationsIndexController extends Controller {
     this.isShowPublicationFilterModal = false;
     this.send('refreshModel');
   }
+
+  @action
+  async getNewPublicationNextNumber() {
+    // Deze query possibly breaks if publication-flows without number exist
+    const publications = await this.store.query('publication-flow', {
```

Comment:
We hebben een uitbreiding van de ember-data story die ook een `this.store.queryOne` voorziet (zie https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/development/app/services/store.js). Dan hoef je de page size niet meer mee te geven en je krijgt automatisch het `firstObject` of `null` indien er geen record is.